cpp/src/jit/cache.cpp

```diff
@@ -74,6 +74,18 @@ boost::filesystem::path getCacheDir()
   // empty, to disallow use of file cache at runtime.
   if (not kernel_cache_path.empty()) {
     kernel_cache_path /= std::string{CUDF_STRINGIFY(CUDF_VERSION)};
+
+    // Make per device cache based on compute capability. This is to avoid multiple devices of
+    // different compute capability to access the same kernel cache.
+    int device;
+    int cc_major, cc_minor;
```

Comment:
I've been told to avoid the "comma operator". One example: https://github.com/rapidsai/cudf/pull/3292#discussion_r357270719
(I have many more examples but cannot find the actual argument against them).

Perhaps this would be a developer guide item.

Reply:
Sure. I'm curious why though. @codereport ?

Reply:
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Res-name-one